src/lib/LUA/lua.cpp

```diff
@@ -289,25 +287,28 @@ bool luaHandleUpdateParameter()
         updateElrsFlags();
         sendELRSstatus();
       } else if (crsf.ParameterUpdateData[1] == 0x2E) {
-        suppressCurrentLuaWarning();
+        luaSupressCriticalErrors();
       } else {
         uint8_t id = crsf.ParameterUpdateData[1];
         uint8_t arg = crsf.ParameterUpdateData[2];
-        // All paramDefinitions are not luaItem_command but the common part is the same
-        struct luaItem_command *p = (struct luaItem_command *)paramDefinitions[id];
-        DBGLN("Set Lua [%s]=%u", p->common.name, arg);
+        struct luaPropertiesCommon *p = paramDefinitions[id];
+        DBGLN("Set Lua [%s]=%u", p->name, arg);
         if (id < LUA_MAX_PARAMS && paramCallbacks[id]) {
-          if (arg == 6 && nextStatusChunk != 0) {
-            pushResponseChunk(p);
+          // While the command is executing, the handset will send `WRITE state=lcsQuery`.
+          // paramCallbacks will set the value when nextStatusChunk == 0, or send any
+          // remaining chunks when nextStatusChunk != nextStatusChunk
```

Comment:
`nextStatusChunk != 0` 😄

Reply:
What I mean is that the comment seems wrong, i.e. `nextStatusChunk != nextStatusChunk` which to me would always be false!

Reply:
🤣 Oh man you're 100% right again. See I say you say != 0 and I knew it was != 0 so I'm all what are you on about?! I didn't even notice the nextStatusChunk != nextStatusChunk. Today is just not my day. Fixed!
Just fried an FC too, I better walk away now.

Reply:
Dude you need a break 😆